### DISCUSSION TITLE
GitHub Copilot keeps automatically turn on autocomplete  even though I have set it off.

### DISCUSSION THREAD

**Discussion by vhbui02** [community/community#126359] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I'm experiencing a weird situation comes from GitHub Copilot VSCode Extension. After an update (I don't know which version), the autocomplete feature is turned on by default without my consent, every time I start a new VSCode instance. I've went inside `settings.json` and set `"github.copilot.editor.enableAutoCompletions": false` numerous times but it keeps toggle on. Why is this happening?

**Comment by tpope** [comment, marked as answer]
Fixed in 1.197.0.
